src/App.tsx

```diff
@@ -1,35 +1,21 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Characters from "./pages/Characters";
 
 function App() {
-  const [count, setCount] = useState(0)
-
+  const client = new ApolloClient({
+    uri: "https://rickandmortyapi.com/graphql",
```

Comment:
This variable is a sensitive data, in a real world application is something must not be visible for any user, in this case you might use a env var instead

Reply:
Yeah, thank you for the suggestions, I will pay more attention to the sensitive data and start using env vars.